client/cache/cache_service_worker.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2017 The Absolute Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+self_.addEventListener('install', (event) => {
+    event.waitUntil(
```

Comment:
Our indentation should be 2 instead of 4.

Reply:
I changed the indent from 4 to 2. I do not know why, what is the reason for 2 indentation?